mypy.ini

```diff
@@ -1,2 +1,3 @@
+# TODO: move to pyproject.toml once python/mypy#5205 is resolved
```

Comment:
we can remove the TODO and track this in a github issue

Reply:
```suggestion
```